src/Common/perf/PerfRunner/project.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": "1.0.0-*",
```

Comment:
The problem isn't that it's using project.json, but that it's pulling down external assets. We'd like for all of the "external dependencies" to be partitioned into the "external" folder in the root. I still think we should move this into buildtools (it's similar to other "Tools programs" we are building there), but if not, then we should make this build similar to the `RemoteExecutorConsoleApp`.